File: 800/469A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int n;
    cin >> n;

    int p;
    cin >> p;
    set<int> levels;

    for (int i = 0; i < p; i++) {
        int a;
        cin >> a;
        levels.insert(a);
    }

    int q;
    cin >> q;
    for (int i = 0; i < q; i++) {
        int a;
        cin >> a;
        levels.insert(a);
    }

    if ((int)levels.size() == n)
        cout << "I become the guy.";
    else
        cout << "Oh, my keyboard!";

    return 0;
}

File: 800/469A/main.java
import java.util.*;

public class main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();  // total number of levels

        int p = sc.nextInt();  // number of levels Little X can pass
        Set<Integer> levels = new HashSet<>();

        for (int i = 0; i < p; i++) {
            levels.add(sc.nextInt());
        }

        int q = sc.nextInt();  // number of levels Little Y can pass
        for (int i = 0; i < q; i++) {
            levels.add(sc.nextInt());
        }

        if (levels.size() == n) {
            System.out.println("I become the guy.");
        } else {
            System.out.println("Oh, my keyboard!");
        }
    }
}
